test/test_client.py

```diff
@@ -0,0 +1,65 @@
+# unit tests for the dts.client package
+
+import dts
+import os
+import unittest
+
+class TestClient(unittest.TestCase):
```

Comment:
I would switch to using pytest for tests -- it is a lot more flexible that unittest as you can easily parametrise your tests (run the same test on a bunch of different inputs) and the test output reporting is much more helpful.

Reply:
Ah, yes, one of the 10M testing packages. Thanks for the recommendation!

Reply:
I made [an issue](https://github.com/kbase/dtspy/issues/6) for this.